src/rustls.rs

```diff
@@ -1,7 +1,8 @@
 #![cfg(feature = "__rustls-used")]
 
 /// Call this before using rustls.
-pub(crate) fn install_default_crypto_provider_if_necessary() -> Result<(), ()> {
+#[allow(clippy::result_unit_err)]
+pub fn install_default_crypto_provider_if_necessary() -> Result<(), ()> {
```

Comment:
issue: This is not intended to be part of the public API of `sspi`.

suggestion: Not ideal, but can you add `#[doc(hidden)]`? Let’s at least hide it.

Reply:
done